gcsfs/tests/conftest.py

```diff
@@ -125,6 +126,48 @@ def gcs(gcs_factory, populate=True):
             pass
 
 
+@pytest.fixture
+def gcs_adapter(gcs_factory, populate=True):
+    # Check if we are running against a real GCS endpoint
+    is_real_gcs = (
```

Comment:
Also a method (actually property) on GCSFileSystem. But actually this condition is wrong: if the env var is missing, we do default to real GCS.